android/core/src/main/java/com/mongodb/stitch/android/core/auth/internal/StitchUserFactoryImpl.java

```diff
@@ -35,8 +37,9 @@ public StitchUser makeUser(
       final String loggedInProviderType,
       final String loggedInProviderName,
       final StitchUserProfileImpl userProfile,
-      final boolean isLoggedIn) {
-    return new StitchUserImpl(
-        id, deviceId, loggedInProviderType, loggedInProviderName, userProfile, auth, isLoggedIn);
+      final boolean isLoggedIn,
+      final Date lastAuthActivity) {
+    return new StitchUserImpl(id, deviceId, loggedInProviderType, loggedInProviderName,
```

Comment:
[minor] does this change also need to be made on the server version of this file?

Reply:
It is made below!